docs/source/developer/bevel_fabric_minikube_setup.md

```diff
@@ -0,0 +1,163 @@
+[//]: # (##############################################################################################)
+[//]: # (Copyright Accenture. All Rights Reserved.)
+[//]: # (SPDX-License-Identifier: Apache-2.0)
+[//]: # (##############################################################################################)
+
+# Running Bevel DLT hyperledger fabric network on minikube
+
+## Pre-requisites
+
+Before proceeding, first make sure that you've completed [developer pre-requisites](./dev_prereq.md).
+
+## Setup minikube
+
+1. Please setup a Ubuntu 20.04 VM with atleast 16 GB RAM, 8 vcpu and having a public ip address, on any cloud provider like Azure, GCP etc. How to set and get a public ip address of VM, one needs to check associated cloud provider documentation. For example: [azure](https://learn.microsoft.com/en-us/azure/virtual-network/ip-services/configure-public-ip-vm)
```

Comment:
For the first step, one could use their own local machine to setup minikube. Can we seperate this into 2 points, 1.1) Getting public IP of local machine and 1.2) Setup on cloud and get the instance public IP